### ISSUE TITLE
whole client crashes when deleting running plot[BUG]

### ISSUE THREAD

**Issue by magnusmyklebost** [Chia-Network/chia-blockchain#4124]
**Describe the bug**

**To Reproduce**

1. Having some running plots;
2. Make new plot
3. Try to delete those new running plots

**Expected behavior**
Should go to "connecting to wallet" forever while CPU usage drops to minimum.

**Screenshots**
![image](https://user-images.githubusercontent.com/36155808/117512709-6e8fc780-af90-11eb-9c3f-b158229bfad1.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: AMD 5900x

**Additional context**
Im new to this and have had 4 parallel plots running for many hours. I wanted to add a queue as it seemed like my 4 parallel plots would be done during night. As I saw the plots I thought would be queued started plotting I pressed delete, and then the client crashed. Have had this happened exactly the same earlier the last time i tried to delete a running plot...

**Comment by PC101guy** [comment]
Happened to me, I am so annoyed, I had 5 plots going and they were like 80% done, added a 6th plot and I was like, I better not push my luck running out of resources so I deleted it and then my screen went from the plot screen to connection to wallet loading forever.... I checked hardware monitor and all drives stopped so no plotting was being done... basically 7 hours down the drain for what?

**Comment by dtuk327** [comment]
Same +1

**Comment by PC101guy** [comment]
> 
> 
> Same +1

Actually now that I think about it, I wasn't actually adding a 6th plot, I just wanted to create ques of my already plotting plots to start up after they finish,  So the first plot going and the one closest to being done 81% was default1, I created my 6th plot as the same name default1 in an attempt for it to be started after the current default1 finished.  However Instead of it going into qued mode it started plotting inwhich I paniced thinking, how is it plotting with the same name as the only already going so i quicking deleted it but that was all it  took to mess up everything.  Hope that makes sense.
